core/src/main/scala/kanaloa/reactive/dispatcher/queue/Worker.scala

```diff
@@ -195,18 +187,22 @@ trait Worker extends Actor with ActorLogging with MessageScheduler {
     startAt:       LocalDateTime = LocalDateTime.now
   ) {
     def success(result: Any): Unit = {
```

Comment:
I have very mixed feelings(leaning towards i don't like) having this internal case class modifying the parent Actor's mutable state.  I don't have much to provide other than subjective reasons.  I'd much prefer to have this be a pure case class which doesn't trigger any side affecting behavior, and have all mutating/state changing code done in the Actor where the rest of it is already located.